### ISSUE TITLE
Build error on Debian 9

### ISSUE THREAD

**Issue by petershaw** [albanm/node-libxslt#62]
```
$ npm --version
3.10.10
```
```
$ node --version
v6.11.2
```

When I am running:  `npm install libxslt` there is the following error: 

```
[...]
make: Entering directory '/var/jenkins_home/jobs/Requester (dev)/workspace/app/node_modules/libxslt/build'
  CC(target) Release/obj.target/libexslt/deps/libxslt/libexslt/common.o
/bin/sh: 1: Syntax error: "(" unexpected
deps/libexslt.target.mk:116: recipe for target 'Release/obj.target/libexslt/deps/libxslt/libexslt/common.o' failed
make: *** [Release/obj.target/libexslt/deps/libxslt/libexslt/common.o] Error 2
[...]
```

I can't find a solution. Can you point me in a direction. I am working on `Linux 22075ab904d4 4.9.0-3-amd64 #1 SMP Debian 4.9.30-2+deb9u3 (2017-08-06) x86_64 GNU/Linux` Debian 9.

**Comment by albanm** [comment]
As it seems to be an environment problem, are you by luck a docker user ?

If so you could you provide a Dockerfile that reproduces the failure ? For example this Dockerfile does not fail:

```
FROM node:8.4.0-alpine
RUN apk add --no-cache make gcc g++ python
RUN npm install libxslt
```

**Comment by petershaw** [comment]
Hi @albanm 
Thanks for pointing me into the right direction. I build several containers and test the installation. it always runs like a charm. But on my Jenkins box it fails. So i rebuild Jenkins (it's docker container already) and it fails again. I checked out the project  in /tmp/project and it installs libxlst! I turned back into my project folder and it fails again... hmmmmm.

The Problem is based in our company rule to name projects. We named the dev branch in Jenkins always with a leading " (dev)" at the end. "_space_bracket" will fail the installation process. 

try the following:

```
 mkdir "foo" && cd "foo" && npm install libxslt;
```
vs.
```
 mkdir "foo (dev)" && cd "foo (dev)" && npm install libxslt;
```

On my mac both works. on Debian 9 the second example fails.
Whooo, this took long to discover ;)
